vars/RedHat_7.yml

```diff
@@ -1,7 +1,4 @@
 # SPDX-License-Identifier: MIT
 ---
-# Put internal variables here with Red Hat Enterprise Linux 7 specific values.
-
-# Example:
-__template_packages: []
-__template_services: []
+__mssql_server_repository: [https://packages.microsoft.com/rhel/7/mssql-server-2019/]
+__mssql_client_repository: [https://packages.microsoft.com/rhel/7/prod/]
```

Comment:
I think the repository should be a string instead of a list.  If you need to handle long strings, use the YAML `>` or `>-` string operator like this:
```yaml
 __mssql_server_repository: >-
  https://packages.microsoft.com/rhel/7/mssql-server-2019/
 __mssql_client_repository: >-
  https://packages.microsoft.com/rhel/7/prod/
```

Reply:
The square brackets were there by default and hence I filled them in. What is the difference between using strings and lists in such cases? In vars/main we used the list ` __mssql_server_packages: [mssql-server]`.